apps/src/code-studio/pd/professional_learning_landing/landingPage.module.scss

```diff
@@ -8,6 +9,26 @@ main.wrapper {
   margin-inline: auto;
 }
 
+section {
+  padding: 4rem 1rem 0 1rem !important;
+}
+
+.headerContainer {
+  background-color: color.$light_gray_50;
+}
+
+.headerWithoutTabsContainer {
+  padding-bottom: 4rem;
+}
+
+.myPlTabsContainer {
+  margin-top: 1.5rem;
+}
+
+.recommendedPlContainer {
```

Comment:
This can be removed since that `<section>` element will adopt the `section` styles you added on line 12. Sorry, I should've been more clear in my original comment!